glib/src/auto/functions.rs

```diff
@@ -539,6 +567,32 @@ pub fn listenv() -> Vec<std::ffi::OsString> {
     unsafe { FromGlibPtrContainer::from_glib_full(ffi::g_listenv()) }
 }
 
+#[doc(alias = "g_locale_to_utf8")]
+pub fn locale_to_utf8(opsysstring: &[u8]) -> Result<(crate::GString, usize, usize), crate::Error> {
+    let len = opsysstring.len() as isize;
+    unsafe {
+        let mut bytes_read = mem::MaybeUninit::uninit();
+        let mut bytes_written = mem::MaybeUninit::uninit();
+        let mut error = ptr::null_mut();
+        let ret = ffi::g_locale_to_utf8(
+            opsysstring.to_glib_none().0,
+            len,
+            bytes_read.as_mut_ptr(),
+            bytes_written.as_mut_ptr(),
+            &mut error,
+        );
+        if error.is_null() {
+            Ok((
+                from_glib_full(ret),
+                bytes_read.assume_init(),
+                bytes_written.assume_init(),
```

Comment:
Isn't this equivalent to the length of the returned `GString`? And I assume it can be used also to give a size hint to it when creating the `GString`